configure

```diff
@@ -500,8 +499,7 @@ cat >"$1/modules.cpp" <<EOT
 name: Qt >= 4.2
 -----END QCMOD-----
 */
-class qc_qt42 : public ConfObj
-{
+class qc_qt42 : public ConfObj {
```

Comment:
I don't know what's you intention in changing auto-generating files but this change  anyway doesn't respect our code style. original code was valid

Reply:
-> https://google.github.io/styleguide/cppguide.html :)

Reply:
What does this document have to do with the psi project?  Project has it's own approved codestyle.

Reply:
-> https://isocpp.org/wiki/faq/coding-standards

Reply:
https://wiki.qt.io/Qt_Coding_Style